### DISCUSSION TITLE
0.3.8 Alpha 3

### DISCUSSION THREAD

**Discussion by BeiDaiQun** [yapeteam/OpenYolBi#5] [Q&A]
拥有了更多1.18.1**Bypass**
**禁止注入网易所有客户端
如有违反则责任全在你**

<hr /><em>This discussion was created from the release <a href='https://github.com/yapeteam/OpenYolBi/releases/tag/'>0.3.8 Alpha 3</a>.</em>

**Comment by BeiDaiQun** [comment]
I need some ideas

**Comment by TIMER-err** [comment]
we need scaffold

**Reply by BeiDaiQun** [reply]
Em, Good idea
